src/contacts/processors/impl/MenuApplicationProcessor.java

```diff
@@ -0,0 +1,18 @@
+package contacts.processors.impl;
+
+import contacts.processors.IActionProcessor;
+
+import java.io.IOException;
+
+public class MenuApplicationProcessor implements IActionProcessor {
+
+    @Override
+    public boolean doAction() throws IOException {
+        return false;
```

Comment:
what for is this processor?

Reply:
This is a requirement of the program for passing the tests